### PR TITLE
Add legend to datapoints

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: client/app/views/contacts/components/datapoints.coffee [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
 
-module.exports = class ContactDatapointsView extends Mn.CollectionView
+module.exports = class ContactDatapointsView extends Mn.CompositeView
 
     tagName: ->
         if @options.cardViewModel.get 'edit' then 'fieldset' else 'ul'
 
+    template: (data) ->
+        name = t "datapoints section #{data.name}"
+        "<legend>#{name}</legend>" if data.edit
```
Comment (aenario) [inline client/app/views/contacts/components/datapoints.coffee]: I had to compile to JS to understand what this function return.
Please make it explicit for both case (data.edit truthy/falsy)  :-)

Comment (m4dz) [inline client/app/views/contacts/components/datapoints.coffee]: Is it better / more understandable if I write it:
```coffee
template: (data) ->
  if data.edit
    name = t "datapoints section #{data.name}"
    "<legend>#{name}</legend>"
  else undefined
```

Comment (aenario) [inline client/app/views/contacts/components/datapoints.coffee]: Yes

Comment (m4dz) [inline client/app/views/contacts/components/datapoints.coffee]: Thanks :+1:!

Comment (frankrousseau) [inline client/app/views/contacts/components/datapoints.coffee]: It think you should declare the variable first and return it at the end of the function using an explicit `return`.

**Pull request merged**